bitnami/argo-cd/templates/_helpers.tpl

```diff
@@ -288,8 +288,13 @@ Argo CD: You need to provide existingSecretPasswordKey when an existingSecret is
 Validate external Redis config
 */}}
 {{- define "argocd.validateValues.externalRedis" -}}
-{{- if not .Values.redis.enabled -}}
+{{- if not .Values.externalRedis.enabled -}}
```

Comment:
This one line would be very problematic because it won't allow deploying with default values (redis subchart):
```
$ helm install --dry-run argo-cd bitnami/argo-cd
Error: INSTALLATION FAILED: execution error at (argo-cd/templates/NOTES.txt:60:4): 
VALUES VALIDATION:
Argo CD: If the redis dependency is disabled you need to add an external redis host
```

My suggestion would be to revert to:
```suggestion
{{- if not .Values.redis.enabled -}}
```

Or alternatively, if we want to consider the value `externalRedis.enabled`, then:
```suggestion
{{- if and .Values.externalRedis.enabled (not .Values.redis.enabled) -}}
```

Reply:
Hey @migruiz4 
Fortunately you discovered the error 👍 

I recently revert the change I made! 

Sorry and thanks!